src/components/toolTip/styles.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import theme from "styles/theme";
+
+import ReactTooltip from "react-tooltip";
+
+export const StyledToolTip = styled(ReactTooltip)`
+    font-size: 1rem;
```

Comment:
let's reduce the copy size to `14px` or even to `12px` and add 1.2 line-height to make it more readable.

Reply:
yes, I set to 12px with 1.2 line-height